### ISSUE TITLE
The nether portal has very weird glitches when the sky is in the backgound

### ISSUE THREAD

**Issue by MrSzalonna** [Despatra/Phoxel#32]
The nether portal disappears when the the sky is behind the portal.
![2024-12-16_15 34 16](https://github.com/user-attachments/assets/cecf8269-530e-4011-a3bd-7815edfb5e32)
GPU: Nvidia RTX 4080
Iris
Thanks!

**Comment by Despatra** [comment]
Thank you for letting me know about this. I'll make sure it's fixed in the next version.

**Comment by Despatra** [comment]
Yeah this is just some bad code, I've got it so it's no longer doing that, so next update will fix the issue.

**Comment by Despatra** [comment]
With the new update, all the code was completely rewritten from scratch, therefore all previous issues are pretty much irrelevant. Because of this I'm closing it. Thank you for the contribution anyways!
